docs/advanced.md

```diff
@@ -38,3 +38,42 @@ The Dashboard panel should now show be showing those APIs:
 Once these steps have been completed, you will be able to use these credentials in the extension.
 In the `jupyterlab.google-drive` settings of the settings registry, set the **clientID** field to be the client id provided by the developer console. If everything is configured properly, you should be able to use the application with your new credentials.
 ![Client ID](images/clientid.png)
+
+### Seeding JupyterLab images with Google credentials
+While adding credentials via the settings functionality from within JupyterLab is possible, as described above, users may also wish to pre-seed these settings so the extension works out-of-the-box on start-up.
+
+With a [redesign](https://github.com/jupyterlab/jupyterlab/pull/2936) of JupyterLab's setting system, the location of settings pertaining to the google-drive extension changed.  When you manually add credentials from within JupyterLab, they now get persisted to:
+
+`home/jovyan/.jupyter/lab/user-settings/@jupyterlab/google-drive/drive.json`
```

Comment:
This is the default location for an image in a premade Jupyter docker container, no? More generally, we can say
"The location of the `@jupyterlab/google-drive` plugin's settings can be found in `$SETTINGS_PATH/@jupyterlab/google-drive/drive.json`, where $SETTINGS_PATH can be found by entering `jupyter lab path` on your terminal."

Reply:
Followed by
"For instance, in the [?] docker image, this path is given by
``home/jovyan/.jupyter/lab/user-settings/@jupyterlab/google-drive/drive.json`
"